src/mon/OSDMonitor.cc

```diff
@@ -1646,6 +1649,8 @@ bool OSDMonitor::check_failure(utime_t now, int target_osd, failure_info_t& fi)
 
   utime_t grace = orig_grace;
   double my_grace = 0, peer_grace = 0;
+  if (propose_now)
+    *propose_now = false;
```

Comment:
hmm, I woudl drop this, and the weird condition when passing the arg or NULL above, and then only set the value to true